packages/@react-spectrum/slider/docs/Slider.mdx

```diff
@@ -0,0 +1,161 @@
+        defaultValue={25} />
+      <Slider
+        label="Cookies to buy (Controlled)"
+        value={value}
+        onChange={setValue} />
+    </Flex>
+  );
+}
+```
+
+Alternatively, a different scale can be used by setting the `minValue` and `maxValue` props.
+
+```tsx example
+<Slider
+  label="Cookies to buy"
+  minValue={50}
+  maxValue={150}
+  defaultValue={100} />
+```
+
+The slider value can be formatted by using the `formatOptions` prop.
+`formatOptions` is compatible with the option parameter of [Intl.NumberFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat) and is applied based on the current locale.
+
+```tsx example
+<Slider
+  label="Currency"
+  formatOptions={{style: 'currency', currency: 'JPY'}}
+  defaultValue={60} />
+```
+
+## Labeling
+Value labels are shown above the Slider by default, but they can also be moved to the side or hidden as needed.
+The label text should be in sentence case.
+
+```tsx example
+<Flex direction="column" maxWidth="size-3000" gap="size-300">
+  <Slider label="Cookies to buy" defaultValue={25} />
+  <Slider label="Donuts to buy" labelPosition="side" defaultValue={25} />
+  <Slider label="Pastries to buy" showValueLabel={false} defaultValue={25} />
+</Flex>
+```
+
+By default, the value label is formatted as a plain number. This can be customized using the following props.
+
+* `showValueLabel`: Allows you to display or hide the value label.
+* `formatOptions`: Allows you to customize the format of the value.
+* `valueLabel`: Allows you to customize the value label to any string.
+
+```tsx example
+function Example() {
+  let [donuts, setDonuts] = React.useState(54);
+  return (
+    <Flex direction="column" maxWidth="size-3000" gap="size-300">
+      <Slider
+        label="Cookies to buy"
+        showValueLabel={false}
+        defaultValue={90} />
+
+      <Slider
+        label="Donuts to buy"
+        maxValue={60}
+        valueLabel={`${donuts} of 60 Donuts`}
+        value={donuts}
+        onChange={setDonuts} />
+
+      <Slider
+        label="Percent donus eaten"
+        maxValue={1}
+        step={0.001}
+        formatOptions={{style: 'percent', minimumFractionDigits: 1}}
+        defaultValue={0.891} />
+    </Flex>
+  );
+}
+```
+
+### Accessibility
+
+Depending on the visualization being used (i.e. depending on the `showValueLabel` prop), a `label`, `aria-label`, or `aria-labelledby` prop is required.
+
+### Internationalization
+
+To internationalize a Slider, a localized string should be passed to the `label` prop, `aria-label` prop, or value of the associated `aria-labelledby` element.
+
+For languages that are read right-to-left (e.g. Hebrew and Arabic), the layout of the slider is automatically flipped. In addition,
+Slider automatically uses the current locale to format the value label.
+
+## Props
+
+<PropTable component={docs.exports.Slider} links={docs.links} />
+
+## Visual options
+
+### Fill
```

Comment:
Color is mentioned in the props documentation of fill so I think we should have an example with color.

Reply:
? I do not follow, fill doesn't have the ability to specify the color.

Reply:
From the props docs "Whether a fill color is shown between the start of the slider and the current value." I now realize that it means the spectrum css fill color, not a user defined color.